appeals/e2e/cypress/e2e/back-office-appeals/addCorrespondence.spec.js

```diff
@@ -16,14 +16,14 @@ describe('Add correspondence', () => {
 	beforeEach(() => {
 		cy.login(users.appeals.caseAdmin);
 	});
-
+	let sampleFiles = caseDetailsPage.sampleFiles;
```

Comment:
I'd recommend having an extra carriage return after this so it's easily to see this is a stand alone line